src/database/models/Payments.ts

```diff
@@ -0,0 +1,30 @@
+import { Schema, model } from "mongoose";
+
+
+
+enum Status {
+    Accepted,
+    Pending,
+    Rejected
+} 
+
+export interface IPayments {
```

Comment:
I think we should make the name non-plural. The name implies that this object holds a list of payments when the object only represents a single payment.